test/launch_local_env.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#assuming we're launched from inside the configuration-anomaly-detection repository
+CAD_REPO_PATH=$(git rev-parse --show-toplevel)
+echo "Assuming CAD repository root is ${CAD_REPO_PATH}"
+
+#loading env vars
+. ${CAD_REPO_PATH}/test/set_stage_env.sh 
+
+#check presence of binary, assume the dnf package name is the same
+check_presence () {
+    # $1 - name of the binary
+    echo -n "Checking presence of $1..."
+    if ! which $1 2>/dev/null >/dev/null; then
+        echo "Not Found"
+        echo "Try 'dnf install $1' on Fedora"
+        exit -1
+    else
+        echo "Found"
+    fi
+}
+
+# clean up child processes on SIGINT
+trap "kill -- -$$" EXIT
+
+check_presence "jq"
```

Comment:
polish - non blocking: can we check those before loading the stage envs to avoid the browser switching/popping until the dependencies are ready?

Mhh or maybe just add a step for the required binaries in the README instructions like 
`dnf install jq openssl tinyproxy haproxy proxytunnel` and it will skip the ones that are already installed